Src/Support/Google.Apis.Core/Util/ExponentialBackOff.cs

```diff
@@ -69,6 +80,25 @@ public ExponentialBackOff(TimeSpan deltaBackOff, int maximumNumOfRetries = 10)
             MaxNumOfRetries = maximumNumOfRetries;
         }
 
+        /// <summary>
+        /// Constructs a new exponential back-off with the given delta percent and maximum retries.
+        /// If unspecified, <paramref name="maximumNumOfRetries"/> is set to 10.
+        /// </summary>
+        public ExponentialBackOff(short deltaBackOffPercent, int maximumNumOfRetries = 10)
```

Comment:
I'd be tempted to make this a factory method with a private constructor - something taking "two integers" could end up getting confusing if we add more overloads later.